src/staging/src/components/park/parkOverview.js

```diff
@@ -37,6 +37,7 @@ export default function ParkOverview({ data: parkOverview, type }) {
   }, [expanded])
 
   const isLong = height >= 260
+  const collapsedParkOverview = parkOverview.split('<a').join('<a tabindex="-1"')
```

Comment:
This would also split on <abbr, <address, <article, <aside, <area.  Can you use something like **cheerio** to do this instead?